packages/slice-machine/lib/models/common/http/DefaultClient.ts

```diff
@@ -197,6 +199,10 @@ export default class DefaultClient {
     return this.trackingFetcher("", review, "", "post");
   }
 
+  async sendOnboarding(data: OnboardingTrackingEvent) {
+    return this.trackingFetcher("", data, "", "post");
```

Comment:
OPT : Wondering if this could be improved because the first and third parameters are always empty string and it's always a post.

Reply:
yeah, I think the idea here was have some generic functions idk